src/transformers/modeling_tf_utils.py

```diff
@@ -303,11 +303,9 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
             elif os.path.isfile(pretrained_model_name_or_path + ".index"):
                 archive_file = pretrained_model_name_or_path + ".index"
             else:
-                archive_file = hf_bucket_url(pretrained_model_name_or_path, postfix=TF2_WEIGHTS_NAME)
-                if from_pt:
-                    raise EnvironmentError(
-                        "Loading a TF model from a PyTorch checkpoint is not supported when using a model identifier name."
-                    )
+                archive_file = hf_bucket_url(
+                    pretrained_model_name_or_path, postfix=(WEIGHTS_NAME if from_pt else TF2_WEIGHTS_NAME)
+                )
```

Comment:
Does this mean that a user could do `TFAutoModel.from_pretrained("xlm-roberta-base", from_pt=True)` and it will load the TF XLM-R model even though there is currently no TensorFlow checkpoint `xlm-roberta-base` on our S3?

Reply:
That's really cool